### ISSUE TITLE
`.ui-state-focus` missing from main navigation in 2.4.5 and 2.4.6

### ISSUE THREAD

**Issue by schizek** [pepe1518/magento2#7]
Accessibility issue introduced in 2.4.5 and remains an issue in 2.4.6. The `ui-state-focus` class is missing on items in the menu in focus preventing styles from being added visually identifying which element is in focus for keyboard only users.  This violates [WCAG Success Criteria 2.4.7 Focus Visible (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html).  The class exists as expected in version 2.4.3-p2.

Steps to reproduce
--
1. Install a vanilla Magento version 2.4.5 - 2.4.6
2. Open developer tools and inspect a submenu item in the main navigation to watch classes
3. Using keyboard tab to the main menu then use arrow keys to navigate to submenu items

Expected result
--
1. When focus is on a menu item a class should be added of `ui-state-focus`

Actual result
--
1. No `ui-state-focus` class is added, just `ui-state-active`

**Comment by schizek** [comment]
Closed, wrong repo.  Please delete.
